### PR TITLE
Documentation fixes.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: samples/androidNativeActivity/README.md [hunk 1]
```diff
@@ -8,5 +8,6 @@ See https://developer.android.com/studio/index.html for more details on Android
 
 To build use `ANDROID_HOME=<your path to android sdk> ../gradlew buildApk`.
 
-Run `$ANDROID_HOME/sdk/platform-tools/adb install -r build/outputs/apk/androidNativeActivity-arm-debug.apk`
+Run `$ANDROID_HOME/platform-tools/adb install -r build/outputs/apk/debug/androidNativeActivity-arm-debug.apk`
```
Comment (SvyatoslavScherbina) [inline samples/androidNativeActivity/README.md]: > `androidNativeActivity-arm-debug.apk`

On my machine it is `androidNativeActivity-debug.apk`

File: samples/csvparser/README.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ or pass `-PrunArgs="./European_Mammals_Red_List_Nov_2009.csv 4 100"` to gradle r
 
 Alternatively you can run artifact directly 
 
-    ./build/konan/bin/CsvParser/CsvParser.kexe ./European_Mammals_Red_List_Nov_2009.csv 4 100
+    ./build/konan/bin/<target>/CsvParser.kexe ./European_Mammals_Red_List_Nov_2009.csv 4 100
```
Comment (SvyatoslavScherbina) [inline samples/csvparser/README.md]: And now these path are incorrect when compiling with `build.sh`.

File: samples/androidNativeActivity/README.md (hunk 1 above)
Comment (olonho) [reply samples/androidNativeActivity/README.md]: OK

File: samples/csvparser/README.md (hunk 2 above)
Comment (olonho) [reply samples/csvparser/README.md]: Absolutely

**Pull request merged**